src/shared/components/ErrorAlert/reducers/errorAlert.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ErrorAlertAction,
+  ErrorAlertAllActions,
+  ErrorAlertState,
+} from '../types/errorAlertTypes';
+
+const errorAlertState: ErrorAlertState = {
+  message: 'Something goes wrong',
+  isOpened: false,
+};
+
+export const errorAlertReducer = (
+  state = errorAlertState,
+  action: ErrorAlertAllActions,
+): ErrorAlertState => {
+  switch (action.type) {
+    case ErrorAlertAction.ERROR_ALERT_TOGGLE:
+      return {
+        ...state,
+        isOpened: !state.isOpened,
+      };
+    case ErrorAlertAction.ERROR_ALERT_SET_ERROR_MESSAGE:
+      return {
+        ...state,
+        isOpened: true,
+        message: action.payload,
```

Comment:
can this logic will be implemented in toggle modal?

Reply:
No :( We take payload from error in thunk actions